app/main/views/users.py

```diff
@@ -76,15 +82,18 @@ def list_users():
 
         user_query = user_query.filter(User.supplier_id == supplier_id)
 
-    users = user_query.paginate(
+    paginated_users = user_query.paginate(
         page=page,
         per_page=current_app.config['DM_API_SERVICES_PAGE_SIZE'],
     )
 
     return jsonify(
-        users=[user.serialize() for user in users.items],
+        users=[user.serialize() for user in paginated_users.items],
+        meta={
+            "total_pages": paginated_users.pages
```

Comment:
Any idea why we don't have a "total" key in our meta like for briefs? I'm guessing it's not been needed but just wondering if there was an explanation for the inconsistency?